_data/changelogs/tokens-color-state.yml

```diff
@@ -2,6 +2,13 @@ title: State color tokens
 type: token
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added `disabled-lighter` and `disabled-darker` tokens.
+    summaryAdditional: Also updated the default values for all disabled tokens.
+    affectsStyles: true
+    githubPr: 5402
+    githubRepo: uswds
+    versionUswds: N.N.N
```

Comment:
Should we add `isBreaking: true` for consistency (its showing on the Settings page)?

**Settings changelog**
![image](https://github.com/uswds/uswds-site/assets/3385219/ba184ef0-2bae-4d8d-8563-8ae052bf593a)

**State color token changelog**
![image](https://github.com/uswds/uswds-site/assets/3385219/da4fbe76-7a8e-4937-b812-8de9f5f52336)

Reply:
Yeah, I think that is probably best. I'm a bit torn because that particular change is not breaking, but it feels safer to let users know that bringing in that update could involve a breaking change.  Updated in [cbb08ee](https://github.com/uswds/uswds-site/pull/2214/commits/cbb08ee106f25eb199726e53384ce807f77b03e4)